google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/TimePartitioningTest.java

```diff
@@ -66,14 +64,22 @@ public void testBuilder() {
 
   @Test
   public void testTypeOf_Npe() {
-    thrown.expect(NullPointerException.class);
-    TimePartitioning.of(null);
+    try {
+      TimePartitioning.of(null);
+      fail();
```

Comment:
It is more readable to keep the class name `Assert.fail()`

Reply:
@rahulKQL only fail or also assertNotNull ?

Reply:
Only for fail, assertNotNull is readable without classname